src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -3868,9 +3868,11 @@ optionalt<symbol_exprt> c_typecheck_baset::typecheck_gcc_polymorphic_builtin(
     }
 
     code_typet t(
-      {code_typet::parametert(ptr_arg.type())}, ptr_arg.type().subtype());
-    t.make_ellipsis();
-    symbol_exprt result(identifier, t);
+      {code_typet::parametert(ptr_arg.type()),
+       code_typet::parametert(ptr_arg.type().subtype()),
+       code_typet::parametert(signed_int_type())},
```

Comment:
This change didn't resolve a test? Add one?

Reply:
Done.